tutorials/lab/rabbit.rst

```diff
@@ -0,0 +1,127 @@
+.. _rabbit:
+
+=================
+Flux with Rabbits
+=================
+
+
```

Comment:
Maybe we could open with a sentence of two that says what a rabbit is, and example cases when I might want to use them (even a bullet list could work here!)

Reply:
That's probably a good idea but I think I'm going to punt on it because (for now) all our users are going to be motivated to try things out and will probably know what they want to use the rabbits for. Besides I think all the examples I can cook up myself would get something wrong...